tests/benchmark/data/serializer.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+from VecSim import *
+import h5py
+import os
+
+# location = os.path.join(os.path.abspath('.'), 'tests', 'benchmark', 'data', '')
+location = os.path.join(os.path.abspath('.'), '')
+print('at: ', location)
```

Comment:
Is this a leftover? 
Explain in a comment what should be the current directory when running this script